pkgs/top-level/all-packages.nix

```diff
@@ -5429,6 +5429,8 @@ with pkgs;
 
   fbvnc = callPackage ../tools/admin/fbvnc {};
 
+  febio-studio = libsForQt5.callPackage ../applications/science/biology/febio-studio {};
```

Comment:
this is probably in the wrong section. there's one for `SCIENCE/BIOLOGY`, but `all-packages.nix` is a giant mess anyways -.-